records/config.go

```diff
@@ -177,14 +177,13 @@ func readConfig(file string) (*Config, error) {
 	c := NewConfig()
 
 	workingDir := "."
-	usr, err := user.Current()
-	if err != nil {
-		// this can happen (on Linux) if you're running mesos-dns as a non-root user.
-		logging.Error.Println("Failed to determine current user, translating ~/ to ./, error was", err)
-	} else {
-		workingDir = usr.HomeDir
+	for _, name := range []string{"HOME", "USERPROFILE"} { // *nix, windows
```

Comment:
That's a lot of env vars to choose from, I suggest to be inspired by the go std pkg which uses the user profile directory (https://github.com/golang/go/blob/release-branch.go1.5/src/os/user/lookup_windows.go#L115).

So I suggest to leave the `USERPROFILE` env var as is.